Pulse/PulseViewController.swift

```diff
@@ -88,7 +88,11 @@ class PulseViewController: UIViewController {
     
     private func toggleTorch(status: Bool) {
         guard let device = AVCaptureDevice.default(for: .video) else { return }
-        device.toggleTorch(on: status)
+        do {
+            try device.toggleTorch(on: status)
+        } catch {
+            print(error)
```

Comment:
Not part of the challenge, but since you're at the view controller now, you could display an alert here 😁